llvm/lib/Transforms/IPO/GlobalOpt.cpp

```diff
@@ -988,6 +988,11 @@ OptimizeGlobalAddressOfAllocation(GlobalVariable *GV, CallInst *CI,
                                InitBool->getName() + ".val", false, Align(1),
                                LI->getOrdering(), LI->getSyncScopeID(),
                                LI->getIterator());
+      // FIXME: Should we use the DebugLoc of the load used by the predicate, or
```

Comment:
I see `new StoreInst` above which looks like it does similar straightforward replacement, should we carry over the sloc there?

Reply:
It certainly looks like it - I'll find a test case and add it (and any other nearby cases if I spot any).